README.md

```diff
@@ -12,86 +12,160 @@ Read more in: [What is this repo?](#what-is-this-repo)
+Some of our clusters do not come with an ingress controller installed by default,
+that way you can pick one of the implementations that best suits your needs.
+This is one of the more popular ingress controllers.
 
-Ingress controllers watch the kubernetes api server's `/ingresses` endpoint for updates to the Ingress resource.
+Ingress controllers watch the kubernetes api server's `/ingresses` endpoint for
+updates to the Ingress resource.
 
 **Table of Contents:**
 
 - [nginx Ingress Controller App](#nginx-ingress-controller-app)
-- [Usage](#usage)
-  - [Sample values files](#sample-values-files)
-  - [Sample App CR and ConfigMap](#sample-app-cr-and-configmap)
-- [Configuration](#configuration)
+- [Installing](#installing)
+  - [Sample values files for the web interface and API](#sample-values-files-for-the-web-interface-and-api)
+  - [Sample App CR and ConfigMap for the Control Plane](#sample-app-cr-and-configmap-for-the-control-plane)
+  - [Important note about required cluster level config](#important-note-about-required-cluster-level-config)
+- [Configuration Options](#configuration-options)
 - [Limitations](#limitations)
 - [For developers](#for-developers)
   - [Installing the Chart locally](#installing-the-chart-locally)
   - [Release Process](#release-process)
 - [What is this repo?](#what-is-this-repo)
 
 
-# Usage
+# Installing
 
+There are 3 ways to install this app onto a tenant cluster.
 
+1. [Using our web interface](https://docs.giantswarm.io/reference/web-interface/app-catalog/)
+2. [Using our API](https://docs.giantswarm.io/api/#operation/createClusterAppV5)
+3. Directly creating the App custom resource on the Control Plane.
 
-## Sample values files
+## Sample values files for the web interface and API
 
 This is an example of the values file you could upload using our web interface.
 
 ```
 # values.yaml
 
-app: hello
-yolo: bolo
+configmap:
+  error-log-level: info
 ```
 
-If you are not using the web interface, our (deprecated) API takes the same structure but formatted as JSON:
+If you are not using the web interface, our (deprecated) API takes the same
+structure but formatted as JSON:
 
 ```
 # values.json
 
 {
-  "app": "hello",
-  "yolo": "bolo",
+  "configmap": {
+    "error-log-level": "info"
+  }
 }
 ```
 
-## Sample App CR and ConfigMap
+## Sample App CR and ConfigMap for the Control Plane
 
-If you have access to the Kubernetes API on the Control Plane, you could create the App CR and ConfigMap directly. Here is an example:
+If you have access to the Kubernetes API on the Control Plane, you could create
+the App CR and ConfigMap directly. Here is an example:
 
 ```
 # appCR.yaml
-
-WIP
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  labels:
+    app-operator.giantswarm.io/version: 1.0.0
+  name: nginx-ingress-controller-app
+  namespace: abc12
```

Comment:
What about highlight this is the cluster ID of the cluster where you want to install it?

Reply:
Ok! I'll make that clear in the opening text and highlight it here too.